modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,97 @@
+= Anypoint Monitoring and Visualizer
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Considerations
+
+=== Stateful sets directories
+
+Each stateful set uses different directories to hold its data, resulting in the following directories:
+
+```
+  /var/lib/data/influxdb/data
+  /var/lib/data/influxdb/meta
+  /var/lib/data/logstash
+  /var/lib/data/dias-cassandra
+  /var/lib/data/dias-meld
+```
+
+InfluxDB meta pods (3 pods) use `/var/lib/data/influxdb/meta` while data pods (2 pods) use `/var/lib/data/influxdb/data`. Logstash uses `/var/lib/data/logstash` and finally Insight uses the remaining two directories to hold its data.
+
+== Port Requirements
+
+Port 8895 needs to be accesible in the main balancer along with the rest of the required ports.
+
+
+=== InfluxDB License
+
+InfluxDB requires a valid license to operate, which should have been provided at the time of purchase of Anypoint Monitoring.
+
+Anypoint Monitoring requires a Kubernetes secret to be present in the cluster before all Anypoint Monitoring/Visualizer charts are deployed.
+
+The following command needs to be run in order to create the required secret in the default namespace:
+
+
+```
+kubectl create secret generic influxdb-license --from-literal=license={license}
+```
+
+Note that InfluxDB license might be a JSON file in which case we only need the `key` value in that secret.
+
+If no secret is given, InfluxDB will fail with the following message
+
+```
+ts=ZULU_TIME lvl=error msg="LICENSE IS INVALID OR CANNOT BE FOUND" log_id=LOG_ID service=licensing error=ERROR_MESSAGE
+```
+
+If that is the case, the secret will need to be created and InfluxDB chart redeployed. Before redeploying InfluxDB, all *pvc*, *pv* and *jobs* (named `dias-prov-k8s-am-influxdb-comp-bootstrap`) will need to be deleted.
+
+== Monitoring known issues
+
+ . Won't be able to create a new Advanced Alert if the sum of Advanced and Basic alerts is >= 20
```

Comment:
@wwarner-salesforce @alejandrorascovan It's okay to add here the **Monitoring known issues** section?

Reply:
Yes, I think so. What do you think @alejandrorascovan ?

Reply:
Oh, I see. I thought this was a section about how users could monitor issues with the cluster that only appear at runtime. Yes, if PCE has known issues regarding monitoring, then those should go in the release notes. Thank you, @alejandrorascovan !

Reply:
@alejandrorascovan Can you point me to the release notes? I can move this over.

Reply:
@juanpablotimpanaro - I moved the dummy RNs to the release-notes repo: 

- branch: DOCS-9705-PCE-3.0-release-notes-hn
- PR: https://github.com/mulesoft/docs-release-notes/pull/3017